### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by carlosms**
Based on #170.
The bullets from #67 fixed in this PR are:

- a612ddc: assume no context/background in the people reading the README.md [#29 (comment)](https://github.com/src-d/code-annotation/pull/29#discussion_r166553539),
- b9c37bc: link to Contributing#Development from our README.md [#29 (comment)](https://github.com/src-d/code-annotation/pull/29#discussion_r166553539)
- 7bd371c: update screenshot

### CODE REVIEW

**Review by dpordomingo — APPROVED** [review]
LGTM.
Thanks for splitting the commits and for making the PR as small as possible.

File: README.md [hunk 1]
```diff
@@ -127,7 +148,7 @@ CAT_OAUTH_RESTRICT_REQUESTER_ACCESS=team:123456
 
 ## Contributing
 
-[Contributions](https://github.com/src-d/code-annotation/issues) are more than welcome, if you are interested please take a look to our [Contributing Guidelines](CONTRIBUTING.md).
+[Contributions](https://github.com/src-d/code-annotation/issues) are more than welcome, if you are interested please take a look to our [Contributing Guidelines](CONTRIBUTING.md). You have more information on how to deploy it for [development purposes here](CONTRIBUTING.md#Development).
```
Comment (smacker) [inline README.md]: @carlosms your English is better than mine. Are you sure `deploy` is the correct word here? For developers, we give instructions how to `run locally` not `deploy`. But I can be wrong.

Comment (bzz) [reply README.md]: `how to deploy it for ...` -> `how to build and run locally for ...` ?

Comment (carlosms) [reply README.md]: You are right, _running locally_ is more clear. Fixed in https://github.com/src-d/code-annotation/pull/177/commits/1bde8b93ddc5a87cf976dfa39b7323d902cb1704

**Comment by bzz** [comment]
@carlosms could you please rebase, as #170 is merged now? 

Might be also worth incorporating latest discussion from https://github.com/src-d/code-annotation/issues/67 before merging.

**Comment by dpordomingo** [comment]
@carlosms now all the feedback seems to be addressed, can you merge?
Many thanks!

**Comment by carlosms** [comment]
@dpordomingo I'm waiting because of the conversation on the issue, we may want to add more things to this PR: https://github.com/src-d/code-annotation/issues/67#issuecomment-369562233

**Review by bzz — APPROVED** [review]

**Pull request merged**